website/settings/defaults.py

```diff
@@ -327,7 +327,7 @@ def parent_dir(path):
 # datacite
 DATACITE_USERNAME = None
 DATACITE_PASSWORD = None
-DATACITE_URL = None
+DATACITE_URL = 'https://mds.datacite.org'
```

Comment:
For non-production environments, should this be the fake datacite url used in tests previously?